src/include/mpir_yaksa.h

```diff
@@ -0,0 +1,13 @@
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ */
+
+#ifndef MPIR_YAKSA_H_INCLUDED
+#define MPIR_YAKSA_H_INCLUDED
+
+int MPII_yaksa_init(void);
+int MPII_yaksa_finalize(void);
```

Comment:
I think these functions can be removed and `yaksa_init/finalize` can be directly called.